layers/meta-resin-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend

```diff
@@ -12,6 +12,21 @@ SRC_URI_append = " \
     file://U-Boot-3-3-config-am335x_evm-detect-BeagleBone-Blue-using-BLA.patch \
     file://0005-Autoboot-keyboard-beaglebone-fixes.patch \
     file://uEnv.txt_internal \
+   "
+
+RESIN_EXTERNAL_MMC_beaglebone-pocket = "1"
+RESIN_INTERNAL_MMC_beaglebone-pocket = "0"
```

Comment:
I think these 2 variables are not used. So we can remove them. Also, since we're at it, let's also remove the ones from the beginning of the file too.